tools/updateFolder.ts

```diff
@@ -0,0 +1,75 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+function updateFileContent(filePath: string, oldFolderName: string, newFolderName: string): void {
+  const content = fs.readFileSync(filePath, 'utf8');
+  const idRegExp = new RegExp(`(id: ${oldFolderName})(.*)(\n)`, 'g');
```

Comment:
Minor
need some logging when topic has "uid" (discovered from sources, and cannot be simply renamed), but not "id"